app/Console/Kernel.php

```diff
@@ -24,8 +24,8 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        // $schedule->command('inspire')
-        //          ->hourly();
+        $schedule->command('scrap:moneyBox')
+                 ->everyFifteenMinutes();
     }
```

Comment:
Tutaj znowu żeby to działało trzebaby dodać do readme informację o konfiguracji tego crona.

Coś w stylu:
https://laravel.com/docs/9.x/scheduling#running-the-scheduler